packages/flutter/lib/src/services/binding.dart

```diff
@@ -47,6 +47,7 @@ mixin ServicesBinding on BindingBase, SchedulerBinding {
     SystemChannels.accessibility.setMessageHandler((dynamic message) => _handleAccessibilityMessage(message as Object));
     SystemChannels.lifecycle.setMessageHandler(_handleLifecycleMessage);
     SystemChannels.platform.setMethodCallHandler(_handlePlatformMessage);
+    PlatformDispatcher.instance.onViewFocusChange = handleViewFocusChanged;
```

Comment:
Can this be 

```suggestion
    platformDispatcher.onViewFocusChange = handleViewFocusChanged;
```

to enable mocking, etc.? That's also what we do elsewhere in the bindings to set callbacks, e.g.

https://github.com/flutter/flutter/blob/b05937d3dc227e608d79cecd015d7882e6f3181d/packages/flutter/lib/src/scheduler/binding.dart#L869-L870